scripts/create-component/templates/component/src/components/{{componentName}}/{{componentName}}.types.ts.hbs

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { ComponentProps, ShorthandProps } from '@fluentui/react-compose/lib/next/index';
+import { ColorTokens, RecursivePartial, FontTokens } from '@fluentui/theme';
+
+/**
+ * {@docCategory {{componentName}} }
```

Comment:
Tested this and it will leave the leading \. Also reviewed the code that is responsible for the docCategory and it has a trim() so extra spaces aren't a concern